tutor/templates/build/openedx/Dockerfile

```diff
@@ -15,12 +15,26 @@ RUN apt update && \
     apt install -y libssl-dev zlib1g-dev libbz2-dev \
     libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
     xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
-ARG PYTHON_VERSION=3.8.12
+# https://github.com/pyenv/pyenv/releases
+# https://www.python.org/downloads/
+ARG PYTHON_VERSION=3.8.15
 ENV PYENV_ROOT /opt/pyenv
-RUN git clone https://github.com/pyenv/pyenv $PYENV_ROOT --branch v2.2.2 --depth 1
+RUN git clone https://github.com/pyenv/pyenv $PYENV_ROOT --branch v2.3.17 --depth 1
 RUN $PYENV_ROOT/bin/pyenv install $PYTHON_VERSION
 RUN $PYENV_ROOT/versions/$PYTHON_VERSION/bin/python -m venv /openedx/venv
 
+<<<<<<< HEAD
+=======
+###### Install Dockerize to wait for mysql DB availability
+FROM minimal as dockerize
+# https://github.com/powerman/dockerize/releases
+ARG DOCKERIZE_VERSION=v0.19.0
+RUN dockerize_url="https://github.com/powerman/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-$(uname -m | sed 's@aarch@arm@')" \
+    && echo "Downloading dockerize from $dockerize_url" \
+    && curl --fail --location --output /usr/local/bin/dockerize $dockerize_url \
+    && chmod a+x /usr/local/bin/dockerize
+
+>>>>>>> c8ea6838 (feat: upgrade to Palm)
 ###### Checkout edx-platform code
```

Comment:
Following the slack thread in regard of the ARM issue, this bit what caused the problem for me, when I just changed it to use what exists in master i.e. this [commit](https://github.com/overhangio/tutor/commit/fa9e86e188f0a70b84f8b688e83a32a65b7bea10), it worked

Reply:
So I didn't open an issue since I guess the issue wouldn't exists if palm to rebase with master

Reply:
:+1: I also resolved the conflict in this PR.